packages/bom/build.gradle.kts

```diff
@@ -81,38 +88,30 @@ val kotlinPlugins = listOf(
   "redakt",
 )
 
-kover {
-  disable()
-}
-
 catalog {
   versionCatalog {
     // map Elide versions
     version("elide.framework", libs.versions.elide.asProvider().get())
     version("elide.plugin", libs.versions.elide.plugin.get())
 
     // map each peer version
-    peers.forEach { alias, (_, version) ->
-      version(alias, version.get())
-    }
-    versionAliases.forEach { (alias, version) ->
-      version(alias, version.get())
-    }
+    peers.forEach { alias, (_, version) -> version(alias, version.get()) }
+    versionAliases.forEach { (alias, version) -> version(alias, version.get()) }
 
     // define Elide build plugin
     plugin("buildtools", "dev.elide.buildtools.plugin").versionRef("elide.plugin")
 
     // define the BOM (this module)
-    library("elide.bom", Elide.group, "bom").versionRef("elide.framework")
+    library("elide.bom", "dev.elide", "bom").versionRef("elide.framework")
 
     // define Elide library aliases
     libraries.forEach { libName ->
-      library("elide.$libName", Elide.group, "elide-$libName").versionRef("elide.framework")
+      library("elide.$libName", "dev.elide", "elide-$libName").versionRef("elide.framework")
     }
```

Comment:
why was this better? not opposed, just curious; before we could tweak one group and easily change this everywhere, which wasn't in use but could be useful later for publishing

Reply:
good catch, this probably resulted from a bad refactoring/debugging related to the `conventions` project, where these constants lived.